src/generator.js

```diff
@@ -38,7 +38,54 @@ const generateDescription = async (page) => {
         return new DescriptionResult(page, null, GenerationStatus.UNKNOWN_ERROR, 'No front matter found');
     }
     if (fronMatter && fronMatter.description) {
+const getFrontMatter = async (page) => {
```

Comment:
Change this method from a function that's inside the method into a function that is declared at the module level.